projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/launchpad-save-modal.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Launchpad Save Modal
+ *
+ * @package automattic/jetpack
+ */
+
+// Feature name.
+const FEATURE_NAME = 'launchpad-save-modal';
+
+wp_enqueue_script(
+	'launchpad-save-modal',
+	plugins_url( 'index.js', __FILE__ ),
+	array(),
+	JETPACK__VERSION,
+	true
+);
+
+wp_localize_script(
```

Comment:
Can we use `add_inline_script` instead of `localize_script`?

Common consensus in the community is that we used `localize_script` because it _used_ to be the only way to hydrate client data from PHP. Now, `add_inline_script` exists, and `localize_script` should primarily be used for, well, localization.
https://developer.wordpress.org/reference/functions/wp_add_inline_script/

Reply:
Totally, it was supposed to be `add_inline_script`, I guess my changes got mixed up. I'll update that

Reply:
`wp_add_inline_script` is a slight improvement over `wp_localize_script` but you have to be sharp with escaping to ensure no CSRF attacks can happen. See an [example of it in action in the Site Editor](https://github.com/WordPress/WordPress/blob/8ea348f6dbfb3907a41a7bf9c92810966437c3eb/wp-admin/site-editor.php#L120-L128).